features/local_links_manager.feature

```diff
@@ -0,0 +1,9 @@
+@app-local-links-manager
+Feature: Local Links Manager
+
+  @local_network
+  Scenario: Healthcheck
+    Given: I am testing "local-links-manager" internally
+    When: I request "/healthcheck"
+    Then: Json is returned
+    And: I should see ""status":"ok""
```

Comment:
I think usually the `:` wouldn't be on these lines (I'm not sure if it works or not). For example: https://github.com/alphagov/smokey/blob/1efb2db3c2867ee3a27a8375279aebbc653eb75a/features/service_manual_publisher.feature#L6-L8